### ISSUE TITLE
Failed to create LXD conjureup1 network bridge: error: Failed to run: dnsmasq --strict-order --bind-interfaces

### ISSUE THREAD

**Issue by gusfon** [conjure-up/conjure-up#942]
## Report

Thank you for trying **conjure-up**! Before reporting a bug please make sure you've gone through this checklist:

- [ ] Is this problem already documented at https://docs.ubuntu.com/conjure-up/en/troubleshoot#common-spell-problems
- [x] Is conjure-up running inside a virtual machine?
- [ ] Is this problem reproducible with `sudo snap refresh conjure-up --edge`?

## Please provide the output of the following commands

```
which juju
which conjure-up
which lxd
juju version
conjure-up --version
lxd --version
lxc config show
cat /etc/lsb-release
```

Please attach tarball of **~/.cache/conjure-up**:

```
tar cvzf conjure-up.tar.gz ~/.cache/conjure-up
```

## Sosreport

Please attach a sosreport:

```
sudo apt install sosreport
sosreport
```

The resulting output file can be attached to this issue.

### MAAS Users
Which version of MAAS?

## Commands ran

Please outline what commands were run to install and execute conjure-up:

**Comment by battlemidget** [comment]
This should be fixed in the latest 2.3/2.2 snaps. Please provide the information we ask for to make sure it is.

**Comment by battlemidget** [comment]
https://github.com/conjure-up/conjure-up/commit/5bbb979c001b681288e32d44a5345aa6944b8c87
